### ISSUE TITLE
[DevTools] Can be edit Brush if is Property is Null

### ISSUE THREAD

**Issue by workgroupengineering** [AvaloniaUI/Avalonia#10659]
**To Reproduce**

1. Create Project form template
2. Replace the content of `MainWindow.axaml' like this:
   ```xaml
    <Border Background="{x:Null} Width="20" Height="20" BorderThickness="1" BorderBrush="Black"/>    
   ```
4. Run
5. Lunch DevTools
6. Select Border
7. Try to edit Background

**Expected behavior**
Click on value show ColorPicker

**Screenshots**

**Desktop (please complete the following information):**

- OS: All
- Version: cbe0bb861a767b7bf7b268bbaa1d89dcf8523aea

**Additional context**
none

cc: @pr8x

**Comment by timunie** [comment]
@robloo I think Color in the ColorPicker should be nullable. What do you think, is it possible to implement it?

**Comment by robloo** [comment]
@timunie Yes, it is theoretically possible but I'm sure you are aware of all the other discussions around this topic. I'm not in favor of doing this at this point. If I were to do it, I would choose the established DatePicker convention and add a nullable SelectedColor alongside the non-nullable Color property. Adding nullability causes more problems than it solves IMO although that is what you choose for the NumberBox.

Also keep in mind the DevTools control is using a color editor to edit a brush. That logically doesn't make sense. You need a brush editor control first to handle this case. ColorView can select colors within a brush but the brush editor needs to better handle null itself.

**Comment by timunie** [comment]
True in this case, as with a ColorPicker you can only have a SolidColorBrush. `Color? SelectedColor` is what I prefer, but okay probably out of issue scope.

**Comment by robloo** [comment]
@workgroupengineering @pr8x Consider using the converter here: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls.ColorPicker/Converters/DoNothingForNullConverter.cs

This can be moved to a more general, shared location as needed. It allows binding non-nullable with nullable properties without breaking things. It was used in the ColorView itself which has an ListBox to display palette colors. ListBox selected item may be null yet it must be bound to the non-nullable Color property of the control.

Further background: 

Also keep in mind the WinUI control is non-nullable and I haven't heard any complaints about this through the lifetime of the control.
